src/components/molecules/OcNotificationMessage/OcNotificationMessage.vue

```diff
@@ -106,40 +110,26 @@ export default {
 </script>
 
 <style lang="scss">
-.oc-notification-message {
-  word-break: break-word;
-}
-
-// TODO: Refactor after removal of uikit
 .oc-notification-message {
   background-color: var(--oc-color-background-default) !important;
-  border: 1px solid transparent;
   cursor: pointer;
-  font-size: 1.25rem;
-  line-height: 1.4;
   margin-top: var(--oc-space-small);
-  padding: 15px;
   position: relative;
+  word-break: break-word;
+  animation: fadeIn 0.3s;
 
-  &-passive {
-    color: var(--oc-color-swatch-passive-default) !important;
-    border-color: var(--oc-color-swatch-passive-default);
-  }
-  &-primary {
-    color: var(--oc-color-swatch-primary-default) !important;
-    border-color: var(--oc-color-swatch-primary-default);
-  }
-  &-success {
-    color: var(--oc-color-swatch-success-default) !important;
-    border-color: var(--oc-color-swatch-success-default);
-  }
-  &-warning {
-    color: var(--oc-color-swatch-warning-default) !important;
-    border-color: var(--oc-color-swatch-warning-default);
+  @keyframes fadeIn {
```

Comment:
I'll extract this into an animation class/SCSS file for reusability and also make it respect the browser's prefers-reduced-motion setting